Convert benefits-cap-calculator to use ERB templates for questions

floehopper opened the pull request.

chrisroos commented:
Apart from my comment about the indenting of content in `content_for :body` this all looks good to me.

floehopper commented:
Since #2116 has been merged, I've rebased this against master, re-run the new version of the script, updated the checksums, and force-pushed. You can see the body with fixed indentation in [this commit](https://github.com/alphagov/smart-answers/commit/b087e6191854fef8d8b1be2a053620e8ee7cf392#diff-7a038166fc51ab318dc6077c847d94f5).
